### ISSUE TITLE
Cannot load

### ISSUE THREAD

**Issue by kengz** [tensorflow/skflow#145]
Running the example `digits.py` gives the error

```
ImportError: cannot import name 'monitors'
```

Once monitor is removed it works normally again. Is monitor packaged with skflow? I'm running skflow v0.1.0

**Comment by terrytangyuan** [comment]
Can you try installing from Github?

**Comment by kengz** [comment]
Just uninstalled and reinstalled from Github, `monitor` can now be imported, but then it throws another error:

```
ImportError: No module named 'cPickle'
```

I've seen somewhere on stackoverflow that python3 doesn't use `cPickle`, but `pickle` instead? How can this be resolved?

**Comment by terrytangyuan** [comment]
Try again. Should be fixed by #143.

**Comment by kengz** [comment]
Problem solved. Thank you.
